### ISSUE TITLE
Editorial - fix broken header & linked WCAG text in 1.4.8

### ISSUE THREAD

**Issue by pday1** [w3c/wcag2ict#969]
1.4.8. 
Looks like it is missing the word substituted text - just jumps straight to the note

Applying SC 1.4.8 Visual Presentation to non-web documents and non-web software
https://w3c.github.io/wcag2ict/#applying-sc-1-4-8-visual-presentation-to-non-web-documents-and-non-web-software

This applies directly as written, and as described in Intent from Understanding Success Criterion 1.4.8, replacing “browser or other user agent” with “user agent, platform software, or other non-web software” in Note 1.
With these substitutions, the notes would read:

Then goes straight to NOTE 1

**Comment by pday1** [comment]
Checked again. Text was correct - word substitution is just for NOTE 1. 

Only issue was link to intent was missing.

**Comment by pday1** [comment]
Now in PR https://github.com/w3c/wcag2ict/pull/971

**Comment by pday1** [comment]
Merged
